aws_lambda_powertools/utilities/data_classes/lambda_function_url_event.py

```diff
@@ -0,0 +1,18 @@
+from aws_lambda_powertools.utilities.data_classes.api_gateway_proxy_event import APIGatewayProxyEventV2
+
+
+class LambdaFunctionUrlEvent(APIGatewayProxyEventV2):
+    """AWS Lambda Function URL event
+
+    Notes:
+    -----
+    For now, this seems to follow the exact same payload as HTTP APIs Payload Format Version 2.0.
+    Certain keys in this payload format don't make sense for function urls (e.g: `routeKey`, `stage`).
+    These keys will have default values that come on the payload, but they are not useful since they can't be changed.
```

Comment:
```suggestion
    Lambda Function URL follows the API Gateway HTTP APIs Payload Format Version 2.0.
    
    Keys related to API Gateway features not available in Function URL use a sentinel value (e.g.`routeKey`, `stage`).
```

Tip: Write first, edit to be more assertive (is or isnt?), and then settle for the shortest with the same meaning.